### ISSUE TITLE
Do not force to write add, drop, and modify when altering a table

### ISSUE THREAD

**Issue by josebailo** [ujjwalguptaofficial/JsStore#321]
## Summary

When using Typescript, the interface `IAlterQuery` force you to set add, drop, and modify properties when altering a table even if you don't need them. I think would be a good idea make them optionally.

### Basic example

Alter the interface from this:

```typescript
export interface IAlterQuery {
  [version: number]: {
    add: TColumns;
    drop: TColumns;
    modify: TColumns;
  };
}
```

To this:

```typescript
export interface IAlterQuery {
  [version: number]: {
    add?: TColumns;
    drop?: TColumns;
    modify?: TColumns;
  };
}
```

### Motivation

This way you could write an alter with just the things you need to add.

**Comment by ujjwalguptaofficial** [comment]
Ok gotcha. It would be great if you can rise a PR - you have already figured out what to do :). I will just copy paste your code lol :rofl:

**Comment by ujjwalguptaofficial** [comment]
fixed in v - 4.5.6
